client/src/test/index.browser.ts

```diff
@@ -0,0 +1,29 @@
+require('mocha/mocha'); // import the mocha web build
```

Comment:
Can this use the import syntax instead or does it have to be require?

Reply:
Just following docs: https://code.visualstudio.com/api/extension-guides/web-extensions#web-extension-tests

Changing it, but may have to revert if it doesnt work with the test suite im trying to get working.